packages/core/src/hydration/event_replay.ts

```diff
@@ -39,6 +40,17 @@ export function withEventReplay(): Provider[] {
       provide: IS_EVENT_REPLAY_ENABLED,
       useValue: true,
     },
+    {
+      provide: ENVIRONMENT_INITIALIZER,
+      useValue: () => {
+        setReplaceJsactionCode((el: RElement) => {
+          if (el.hasAttribute('jsaction')) {
+            el.removeAttribute('jsaction');
```

Comment:
It'd be great to extract the string into a const and use it here and in other places where we use `jsaction` as an attribute name (during serialization?). That should help avoid typos and provide a reference for additional docs.
```suggestion
          if (el.hasAttribute(JSACTION_ATTR)) {
            el.removeAttribute(JSACTION_ATTR);
```

```
/**
 * Name of the attribute that is added to elements during SSR to capture
 * information about events that were defined in a template. This is a part
 * of event replay logic powered by JSAction. See additional information at:
 * <link-to-jsaction-readme>.
 */
export const JSACTION_ATTR = 'jsaction';
```

Reply:
imported the symbol from event-dispatch :)